frontend/src/v5/store/tickets/tickets.sagas.ts

```diff
@@ -166,7 +166,7 @@ export function* updateTicket({ teamspace, projectId, modelId, ticketId, ticket,
 	}
 }
 
-export function* createTicket({ teamspace, projectId, modelId, ticket, isFederation, onSuccess }: CreateTicketAction) {
+export function* createTicket({ teamspace, projectId, modelId, ticket, isFederation, onSuccess, onError }: CreateTicketAction) {
```

Comment:
- When trying to change a unique property it sends the request twice
- It also tries sending it each time I change focus in the form 
![triple sent](https://github.com/3drepo/3drepo.io/assets/42032433/8beb0736-a484-43d1-b71e-be74957f0ef3)